src/vs/editor/common/config/commonEditorConfig.ts

```diff
@@ -659,11 +659,6 @@ const editorConfiguration: IConfigurationNode = {
 		},
 		'editor.foldingStrategy': {
 			'type': 'string',
```

Comment:
should be changed to
```ts
{ "oneOf": [
	{
	     'type': 'string',
	},
	{
		'type': 'string',
		'enum': ['auto', 'indentation'],
		'enumDescriptions': [
			nls.localize('foldingStrategyAuto', 'If available, use a language specific folding strategy, otherwise falls back to the indentation based strategy.'),
				nls.localize('foldingStrategyIndentation', 'Always use the indentation based folding strategy')
		],
	}

]}
```
so that the proposals are still available.